fenix/app/src/main/java/org/mozilla/fenix/FeatureFlags.kt

```diff
@@ -55,7 +55,8 @@ object FeatureFlags {
     /**
      * Enables compose on the top sites.
      */
-    const val composeTopSites = false
+    // const val composeTopSites = false
+    const val composeTopSites = true
```

Comment:
This will require reverting back to false before landing. For now it is true to check tests.